snuba/migrations/snuba_migrations/system/0001_migrations.py

```diff
@@ -2,12 +2,14 @@
 from snuba.clusters.storage_sets import StorageSetKey
 from snuba.migrations import migration
 from snuba.migrations import operations
+from snuba.migrations.context import Context
+from snuba.migrations.status import Status
 
 
-class Migration(migration.Migration):
+class Migration(migration.CustomMigration):
```

Comment:
Please add a docstring that explains why this migration is special and cannot be a multi-step migration.